### DISCUSSION TITLE
please re-enable coverity builds

### DISCUSSION THREAD

**Discussion by chipitsine** [ImageMagick/ImageMagick#6711] [Contribute]
there were about 10 issues found by coverity

also, coverity badge would be nice

as for implementation, it might be scheduled daily (or weekly) task in travis-ci

**Comment by dlemstra** [comment]
We are planning to move the whole build process over to Azure DevOps. We could kick off a covertity build there. And then add the badge.

**Comment by chipitsine** [comment]
not sure Azure DevOps allows scheduled builds. I need to have a look :)


there are about 10 builds a day in general. no need to kick coverity on each build
